samples/bluetooth/central_uart/README.rst

```diff
@@ -27,10 +27,10 @@ Requirements
 
 Building and running
 ********************
+.. |sample path| replace:: :file:`samples/bluetooth/central_uart`
 
-This sample can be found under :file:`samples/bluetooth/central_uart` in the |NCS| folder structure.
+.. include:: ../../../includes/build_and_run.txt
```

Comment:
Is it worth investigating if we can use a root folder here instead of going down the tree? something like `$RST_ROOT/`

Reply:
That would be really good, yes!
The specs say "The directive argument is the path to the file to be included, relative to the document containing the directive." though, so not sure this will work.

Reply:
Updated.